pages/status.tsx

```diff
@@ -26,6 +27,15 @@ const initialValues: CheckStatusRequestBody = {
 const Status: FC = () => {
   const { t } = useTranslation('status')
 
+  const getLinkSummaryItems: any = () => {
+    const LSI: any = t('no-match-status-links', { returnObjects: true })
+    const noMatchHref: LinkSummaryItem[] = []
+    for (const keys in LSI) {
+      noMatchHref.push(LSI[keys] as LinkSummaryItem)
+    }
+    return noMatchHref
+  }
```

Comment:
I don't think all this should be necessary, would just need to included `t('no-match-status-links', { returnObjects: true })` as the prop for the links...